### DISCUSSION TITLE
Where are the docs/samples?

### DISCUSSION THREAD

**Discussion by AArnott** [WildGums/Orc.Squirrel#376] [Q&A] [answered]
I can't find the docs or samples anywhere. What does integration with a WPF app look like?

**Comment by GeertvanHorrik** [comment, marked as answer]
Sorry, not sure why we missed this question.

The example code should be located in https://github.com/WildGums/Orc.Squirrel/blob/develop/src/Orc.Squirrel.Example/ViewModels/MainViewModel.cs

This example does not include the packaging itself since that is Squirrel generic and out of scope for this project.

**Reply by AArnott** [reply]
🏅 You win the necromancer award.
